packages/paste-core/components/toast/stories/index.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
+import {action} from '@storybook/addon-actions';
+import {Button} from '@twilio-paste/button';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Toast, ToastContainer} from '../src';
+
+storiesOf('Components|Toast', module)
+  .addDecorator(withKnobs)
+  .add('Neutral', () => {
+    return (
+      <ToastContainer>
```

Comment:
It looks liek you're going to have set a box around these with a min height for the snapshots to work. Because the container is fixed, the story doesn't actually have any height

![image](https://user-images.githubusercontent.com/368249/93541089-5ccab880-f90a-11ea-9d3e-354c77dcd6c6.png)

Reply:
Good idea on the minHeight. Looks like that works.